provider/provider.go

```diff
@@ -1564,6 +1564,15 @@ func createUpdateSystemSettingsCommand(systemSettings map[string]interface{}) st
 			builder.WriteString(strconv.Itoa(v))
 		case bool:
 			builder.WriteString(strconv.FormatBool(v))
+		case []interface{}:
+			encodedList, err := json.Marshal(v)
+
+			if err == nil {
```

Comment:
what does this do? this looks like a fallback if the json fails. why would the json encoding fail?

Reply:
Good point - now that I look at it, the json.Marshal wouldn't fail because the settings are defined as string_list and that can always be encoded to string. I've changed this part to:
```go
case []interface{}:
	encodedList, _ := json.Marshal(v)

	builder.WriteString(string(encodedList))